### PR TITLE
Branch a

### PR DESCRIPTION
**Pull request by Namithajacob**

### CODE REVIEW

**Comment by mitalraithatha** [comment]
In Branch A --- 'Vendor' is the 3rd party folder so it should come under the 'lib' folder.

**Comment by mitalraithatha** [comment]
Cadmium and DESTimes folders under lib folder is empty.No content and files are there.
Need to put that folders(Cadmium and DESTimes ) with files and folder content.

File: makefile [hunk 1]
```diff
@@ -0,0 +1,35 @@
+$(shell mkdir -p bin)
+$(shell mkdir -p build)
+
+CC=g++
+CFLAGS=-std=c++17
+
+INCLUDECADMIUM=-I ../../lib/cadmium
```
Comment (SREEJITH1991) [inline makefile]: correct the path to include **cadmium header files** from the **lib** project folder.

**Review by SREEJITH1991 — CHANGES_REQUESTED** [review]
please correct the input and output file paths to the corresponding /data folder location
changes are to be included in the following files

1. main.cpp of receiver
2. main.cpp of sender
3. main.cpp of subnet
4. main.cpp of top_model

**Review by mitalraithatha — CHANGES_REQUESTED** [review]
In Branch A --- 'Vendor' is the 3rd party folder so it should come under the 'lib' folder.

**Review by mitalraithatha — CHANGES_REQUESTED** [review]
Cadmium and DESTimes folders under lib folder is empty.No content and files are there.
Need to put that folders(Cadmium and DESTimes ) with files and folder content.

**Review by mitalraithatha — CHANGES_REQUESTED** [review]
1) With the folder changes, you need to change the respective path as well in total 4 files.
You need to change respective file path under this folder as well : src->top_model_main.cpp.
2) Also you need to update Readme file according to folder changes of lib folder.

**Review by mitalraithatha — APPROVED** [review]
Approve-Changes done properly for vendor folder.

**Review by mitalraithatha — APPROVED** [review]
Approve-All changes done properly for 3rd party folder and also changes done properly in README file as well as for path changes for all four main.cpp.

File: makefile (hunk 1 above)
Comment (dilsgeorge) [reply makefile]: suggested changes in  the paths has been made, please review  he request again

Comment (SREEJITH1991) [reply makefile]: issue resolved--verified

File: test/src/receiver/main.cpp [hunk 2]
```diff
@@ -49,7 +49,7 @@ int main(){
   auto start = hclock::now(); //to measure simulation execution time
 
 /*************** Loggers *******************/
-  static std::ofstream out_data("receiver_test_output.txt");
+  static std::ofstream out_data("../../data/receiver/receiver_test_output.txt");
```
Comment (SREEJITH1991) [inline test/src/receiver/main.cpp]: Discrepancy in the path still exist.. please run your executable and verify if the output files are getting changed. I checked  tried to run via eclipse and it worked when i gave the path as "test/data/receiver/receiver_test_output.txt".
This issue exists for all the input, output file paths of sender,receiver,topmodel and subnet.
Please verify

**Comment by dilsgeorge** [comment]
Suggested changes are made, please review the same

**Review by SREEJITH1991 — APPROVED** [review]
approved

**Pull request merged**